portfolio/src/main/java/com/google/sps/servlets/Comments.java

```diff
@@ -0,0 +1,134 @@
+package com.google.sps.servlets;
+
+import javax.servlet.http.HttpServletRequest;
+
+import com.google.gson.Gson;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.KeyFactory;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.HashMap;
+
+public class Comments {
+
+  /**
+  * Gets comments from datastore at specified ID
+  */
+  public static String getComments(String collectionID) {
+    // Create datastore query
+    Query query = new Query(collectionID).addSort("timestamp", SortDirection.DESCENDING);
+
+    // Prepare database to be queried
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    // Get query results
+    PreparedQuery results = datastore.prepare(query);
+
+    Map<String, List<List<String>>> comments = new HashMap<>();
```

Comment:
Why are comments a List of Lists? If intentional please add a comment in code explaining.

Also, rename to commentsMaps so it's easy to know this is a map when using.

Reply:
I used a list of lists because I want to be able to group comments together. A list which is made up of the comment data is then stored in a list of other comments. I'm grouping comments along with all the other comments that are replies to it.

Reply:
I'll be sure to rename it.